Update Snowflake OAuth's subdomain migration details (#9300)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/set-up-snowflake-oauth.md

## Subdomain migration

> If your account has been assigned a static subdomain for the [access URL migration](/docs/platform/about-platform/account-url-migration), you may need to update Snowflake security integrations shared across projects or connections to prevent service disruptions. Connections created after your account's migration require an additional OAuth redirect URI. You can find this redirect URI under the [connection's OAuth details](#locate-the-redirect-uri-value), or you can reference [API access URLs](/docs/dbt-apis/overview). Once found, alter your existing security integrations to add the redirect URI.

Reviewer: ```suggestion
If your account uses a static subdomain for the [access URL migration](/docs/platform/about-platform/account-url-migration), you might need to update Snowflake security integrations shared across projects or connections to prevent service disruptions. 

Connections created after migration require an additional OAuth redirect URI. Find this redirect URI under the [connection's OAuth details](#locate-the-redirect-uri-value) or in [API access URLs](/docs/dbt-apis/overview), then add it to your existing Snowflake security integration. 
```
